org-asterion/modules/update_stack.py

```diff
@@ -37,13 +37,32 @@ def update_stack_account(self):
         # Define an inline policy document in the asterion stack account for resource permissions
         self.resource_policy = aws.iam.get_policy_document(
             statements=[
+
+                # Allow open permissions across common aws services
+                aws.iam.GetPolicyDocumentStatementArgs(
+                    actions=[
+                        "cloudwatch:*",
```

Comment:
Why these resources?  

ec2 makes sense as we are actively using that for `infra-aws` and cloudwatch for logs sure but do we need `s3` and `dynamodb` at the moment?  If we aren't using them as part of `infra-aws` we could remove it.